tests/docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.0.0
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.16.0
```

Comment:
I think it is important to also run tests against elasticsearch 8. Want me to add them ?

Reply:
Yes please, do so 🙏🏾